src/bit_machine/mod.rs

```diff
@@ -366,66 +367,79 @@ impl BitMachine {
     }
 
     fn exec_jet<J: Jet>(&mut self, jet: J, env: &J::Environment) -> Result<(), JetFailed> {
-        use simplicity_sys::c_jets::frame_ffi::{c_readBit, c_writeBit, CFrameItem};
-        use simplicity_sys::c_jets::uword_width;
-        use simplicity_sys::ffi::UWORD;
+        use crate::ffi::c_jets::frame_ffi::{c_readBit, c_writeBit, CFrameItem};
+        use crate::ffi::c_jets::uword_width;
+        use crate::ffi::ffi::UWORD;
+
+        /// Return a read frame that is equivalent to the active read frame.
```

Comment:
In 494a232fa8e6144be845940a1bfe22c88e98359e:

I'm much happier now that this lives inside `exec_jet` and is clearly a single-use function. But we should improve the comment still, to say something like
```
/// Creates a new read frame, backed by `buffer`, and copy the `bit_width` bits from the active read frame into it.
///
/// The value of `buffer`, including possibly its backing store, will be lost and no pointers
/// to it should be retained. Similarly the returned frame should not outlive `buffer` or
/// any modifications to it.
///
/// `bit_width` should be the width of the active read frame.
```